app/controllers/users_controller.rb

```diff
@@ -1,4 +1,24 @@
 class UsersController < ApplicationController
   def new
+  	@user = User.new
   end
+
+  def show
+    @user = User.find params[:id]
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
```

Comment:
From now, you should implement I18n(internationalization) in your app. 
You can study more from [http://guides.rubyonrails.org/i18n.html](http://guides.rubyonrails.org/i18n.html)